### PR TITLE
Update to README and BUILDING

### PR DESCRIPTION
**Pull request by GPUCode**
Hello, newbie here. Trying to contribute to an emulation project in order to learn about emudev. Made the README a bit more stylish and added some images to demonstrate the emulator resolving issue #574. Also improved the BUILDING guide, which does not address issue #554 because it is for windows, but it addresses some building issues I experienced. Also resolved issue #586 which means the emulator does not crash when launching a game from the recent files list that does not exist.

Καταγραφή

### CODE REVIEW

File: src/decaf-qt/src/mainwindow.cpp [hunk 1]
```diff
@@ -169,7 +184,7 @@ MainWindow::loadFile(QString path)
    mDecafInterface->startLogging();
    mRenderWidget->startGraphicsDriver();
    mDecafInterface->startGame(path);
-
+
```
Comment (CodingKoopa) [inline src/decaf-qt/src/mainwindow.cpp:187]: Unneeded spacing.

File: src/decaf-qt/src/mainwindow.cpp [hunk 2]
```diff
@@ -179,7 +194,7 @@ MainWindow::loadFile(QString path)
       while (files.size() > MaxRecentFiles) {
          files.removeLast();
       }
-
+
```
Comment (CodingKoopa) [inline src/decaf-qt/src/mainwindow.cpp:197]: Ditto.

Comment (GPUCode) [reply src/decaf-qt/src/mainwindow.cpp]: Fixed

File: src/decaf-qt/src/mainwindow.cpp (hunk 1 above)
Comment (GPUCode) [reply src/decaf-qt/src/mainwindow.cpp]: Fixed

File: src/decaf-qt/src/mainwindow.cpp [hunk 3]
```diff
@@ -13,6 +13,7 @@
 #include <QFileDialog>
 #include <QInputDialog>
 #include <QSettings>
+#include <QFileInfo.h>
```
Comment (VelocityRa) [inline src/decaf-qt/src/mainwindow.cpp:16]:
```suggestion
#include <QFileInfo>
```
this breaks linux CI

File: src/decaf-qt/src/mainwindow.cpp [hunk 4]
```diff
@@ -148,7 +149,21 @@ MainWindow::debugInterrupt()
...
+      auto settings = QSettings {};
+      auto files = settings.value("recentFileList").toStringList();
+      files.removeAll(path);
+      settings.setValue("recentFileList", files);
+
+      updateRecentFileActions();
+      return false;
+   }
+
+    // You only get one chance to run a game out here buddy.
```
Comment (dracc) [inline src/decaf-qt/src/mainwindow.cpp:166]: Indentation change for no reason?
Also, does the message still hold true?

**Pull request closed without merging**